atc/db/migration/migrations/1627322253_drop_build_id_from_resources.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE resources ADD COLUMN build_id bigint REFERENCES builds (id) ON DELETE SET NULL;
```

Comment:
This should be dropping the other columns added in the up migration (`in_memory_build_*`)